python3/bin/xe-reset-networking

```diff
@@ -12,6 +12,8 @@ but WITHOUT ANY WARRANTY; without even the implied warranty of
 MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 GNU Lesser General Public License for more details.
 """
+# pyright: reportPossiblyUnboundVariable=false
```

Comment:
Why is it not worth declaring the variable instead?

Reply:
@psafont: Initially, it was my recommendation to not touch the warnings (disable them) because the script's style is quite old (bad):

The reason was that I did not immediately see a clean way to fix all possibly unbound variables.

The causes for these possibly unbound variables is that the exception handling wasn't done nicely.

Many locations trigger this warning, and to fix it, we should use a context manager handling it:

- In case you like it, I've pushed the fixes for `pyright` now to the PR. Please check!